listenbrainz/webserver/static/js/src/year-in-music/YearInMusic.tsx

```diff
@@ -151,6 +151,16 @@ export default class YearInMusic extends React.Component<
             newTrack.extension[MUSICBRAINZ_JSPF_TRACK_EXTENSION] =
               track.extension[badPlaylistTrackExtensionValue];
           }
+          const trackExtension =
+            newTrack?.extension?.[MUSICBRAINZ_JSPF_TRACK_EXTENSION];
+          // See https://github.com/metabrainz/listenbrainz-server/pull/1839 for context
+          if (trackExtension && has(trackExtension, "artist_mbids")) {
+            //  Using some forbiddden (but oh so sweet) ts-ignore as we're fixing objects that don't fit the expected type
+            // @ts-ignore
+            trackExtension.artist_identifiers = trackExtension.artist_mbids;
+            // @ts-ignore
+            delete trackExtension.artist_mbids;
+          }
```

Comment:
Protect the legacy !

A summary of the generated playlists is stored in the year-in-music table in the database, and with the change of JSPF key name we need to ensure we don't break the playlist cards on the YIM page.

Consider re-writing the playlists in the YIM table to use `artist_identifiers` instead of `artist_mbids` and nuking this code block in the future.